### ISSUE TITLE
Can you provide screenshots inside the readme?

### ISSUE THREAD

**Issue by TomerGamerTV** [UnamSanctam/UnamWebPanel#266]
Can you put some screenshots inside the readme to show how it looks like?

**Comment by UnamSanctam** [comment]
I could but since it's a web page the image would have to be cropped in some good way.

**Comment by TomerGamerTV** [comment]
> I could but since it's a web page the image would have to be cropped in some good way.

idk what browser you use but if its firefox i know there is a way to screenshot the whole webpage

**Comment by UnamSanctam** [comment]
Yes but I mean the images would be too large to look any good so I would need to crop it in some good way, unless I were to add a separate folder for screenshots but it doesn't really matter much what it looks like anyways.
